rules/xcodeproj.bzl

```diff
@@ -845,6 +859,7 @@ https://www.rubydoc.info/github/CocoaPods/Xcodeproj/Xcodeproj/Constants
         "installer": attr.label(executable = True, default = Label("//tools/xcodeproj_shims:installer"), cfg = "host"),
         "build_wrapper": attr.label(executable = True, default = Label("//tools/xcodeproj_shims:build-wrapper"), cfg = "host"),
         "additional_files": attr.label_list(allow_files = True, allow_empty = True, default = [], mandatory = False),
+        "additional_prebuild_script": attr.string(default = "", mandatory = False),
```

Comment:
additonal scripts always ran before the Build with bazel script right? If so I think it's worth calling out in the description here